deepface/modules/recognition.py

```diff
@@ -292,7 +295,7 @@ def find(
     return resp_obj
 
 
-def __list_images(path: str) -> list:
+def __list_images(path: str) -> list[str]:
```

Comment:
This line causes failing of unit tests: https://github.com/serengil/deepface/actions/runs/7993768781/job/21857992920?pr=1031

This should be changed to List[str] if you insist to use string here.

Reply:
Thank you. Done.